modules/react/loading-dots/lib/LoadingDots.tsx

```diff
@@ -1,73 +1,75 @@
 import * as React from 'react';
-import {keyframes} from '@emotion/react';
-import {canvas, borderRadius} from '@workday/canvas-kit-react/tokens';
-import {styled} from '@workday/canvas-kit-react/common';
+import {base, system} from '@workday/canvas-tokens-web';
+import {createComponent} from '@workday/canvas-kit-react/common';
+import {createStyles, handleCsProp, keyframes, CSProps, px2rem} from '@workday/canvas-kit-styling';
+
+/**
+ * Duration of the sparkle dots loading (in ms).
+ */
+const ANIMATION_DURATION_MS = '1400';
 
 /**
  * Keyframe for the dots loading animation.
  */
-const keyframesLoading = keyframes`
-  0%,
-  80%,
-  100% {
-    transform: scale(0);
-}
-  40% {
-    transform: scale(1);
-}`;
+const keyframesLoading = keyframes({
+  '0%, 80%, 100%': {
+    transform: 'scale(0)',
+  },
+  '40%': {
+    transform: 'scale(1)',
+  },
+});
 
-type LoadingDotProps = {
-  /**
-   * The animation delay of the LoadingDots (in ms).
-   */
-  animationDelay: number;
-};
+const loadingDotStyles = createStyles({
+  backgroundColor: base.soap400,
+  width: system.space.x4,
+  height: system.space.x4,
+  fontSize: system.space.zero,
+  borderRadius: system.shape.round,
+  transform: 'scale(0)',
+  display: 'inline-block',
+  marginRight: system.space.x2,
+  animationName: keyframesLoading,
+  animationDuration: `${ANIMATION_DURATION_MS}ms`,
+  animationIterationCount: 'infinite',
+  animationTimingFunction: 'ease-in-out',
+  animationFillMode: 'both',
+  gap: px2rem(1),
```

Comment:
`gap` should be `system.space.x2`, `display` should be `inline-flex`, and `marginRight` should be removed.

Reply:
Oops. I mean these should be added to the container styles.

Reply:
Sorry, for confusion will update it